instagram/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "local-storage": "^1.4.2",
```

Comment:
You won't need to bring in a local storage library to add things to local storage.